desc/equilibrium/equilibrium.py

```diff
@@ -564,7 +565,7 @@ def optimize(
             Objective function to satisfy. Default = fixed-boundary force balance.
         optimizer : str or Optimizer (optional)
             optimizer to use
-        ftol, xtol, gtol : float
+        ftol, xtol, gtol, ctol : float
```

Comment:
probably a dumb question but ctol is the tolerance on satisfying the constraints given? So if we give `F balance < 1e-2` as the nonlinear constraint, the `ctol` would hit if we have `F balance` change by `1e-2` or less?

Reply:
no, `ctol` is a criteria on the infinity norm (max abs) of the constraint violation, not a relative tolerance like `ftol`

So if your constraint is `F < 1e-2` and `ctol = 1e-4` then it would really stop when `F < (1e-2 + 1e-4)`